src/build/content/prerendered.ts

```diff
@@ -172,7 +172,7 @@ export const copyPrerenderedContent = async (ctx: PluginContext): Promise<void>
 
               // Netlify Forms are not support and require a workaround
               if (value.kind === 'PAGE' || value.kind === 'APP_PAGE') {
-                verifyNoNetlifyForms(ctx, value.html)
+                verifyNetlifyForms(ctx, value.html)
```

Comment:
I made these verifications all positive because it was getting confusing knowing when to warn